tests/infer/test_inference.py

```diff
@@ -295,6 +295,16 @@ def test_elbo_reparameterized(self):
     def test_elbo_nonreparameterized(self):
         self.do_elbo_test(False, 10000, Trace_ELBO())
 
+    # these "vectorized" tests are used to detect bugs similar to
+    # the one in https://github.com/pytorch/pytorch/issues/9521
+    def test_elbo_reparameterized_vectorized(self):
+        self.do_elbo_test(True, 5000, Trace_ELBO(num_particles=2, vectorize_particles=True,
+                                                 max_iarange_nesting=1))
+
+    def test_elbo_nonreparameterized_vectorized(self):
```

Comment:
sorry could you add a comment to each function? "these" is ambiguous and becomes more ambiguous as more tests are added.

Reply:
yup, I'll make the change :)